modules/oe_migration_workflow/tests/src/Unit/Plugin/migrate/process/SetWorkflowStateTest.php

```diff
@@ -205,6 +205,19 @@ public function testInvalidArgumentConfiguration(array $config, $message) {
     $this->initializePlugin($config);
   }
 
+  /**
+   * Test with an invalid workflow name.
+   */
+  public function testInvalidWorkflowName() {
+    $invalid_workflow_name = $this->getRandomGenerator()->string();
+
+    $configuration = ['workflow_config_name' => $invalid_workflow_name];
+
+    $this->expectException(\InvalidArgumentException::class);
+    $this->expectExceptionMessage(sprintf('"%s" is not a valid workflow.', $invalid_workflow_name));
+    $this->initializePlugin($configuration);
+  }
+
```

Comment:
@upchuk Missing test was restored. Thanks for your feedback.
Cc.: @ioannis-ampatzis